cmd/operator-sdk/generate/packagemanifests/packagemanifests.go

```diff
@@ -0,0 +1,263 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package packagemanifests
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+
+	genutil "github.com/operator-framework/operator-sdk/cmd/operator-sdk/generate/internal"
+	gencsv "github.com/operator-framework/operator-sdk/internal/generate/clusterserviceversion"
+	"github.com/operator-framework/operator-sdk/internal/generate/collector"
+	genpkg "github.com/operator-framework/operator-sdk/internal/generate/packagemanifest"
+	"github.com/operator-framework/operator-sdk/internal/scaffold/kustomize"
+)
+
+//nolint:lll
+const examples = `
+	# Generate manifests then create the package manifests base:
+  $ make manifests
+  /home/user/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+  $ operator-sdk generate packagemanifests -q --kustomize
+
+  Display name for the operator (required):
+  > memcached-operator
+  ...
+
+  $ kustomize build config/packagemanifests | operator-sdk generate packagemanifests --manifests --metadata --overwrite --version 0.0.1
+  Generating package manifests version 0.0.1
+  ...
+
+  # After running the above commands, you should see:
+  $ tree config/packages
+  config/packages
+  ├── bases
+  │   └── memcached-operator.clusterserviceversion.yaml
```

Comment:
That's odd, is `--update-crds` meant to be on by default? I'm seeing that crd manifest as well.
```console
$ kustomize build config/packagemanifests | operator-sdk generate packagemanifests --manifests --version 0.0.1
Generating package manifests version 0.0.1
Package manifests generated successfully in config/packagemanifests

$ tree config/packagemanifests/
config/packagemanifests/
├── 0.0.1
│   ├── cache.example.com_memcacheds.yaml
│   └── memcached-operator.clusterserviceversion.yaml
├── bases
│   └── memcached-operator.clusterserviceversion.yaml
├── kustomization.yaml
└── memcached-operator.package.yaml
```

Reply:
Argh, wrong line. Going to comment down below. Ignore this.